morceus-data/latin/stems/verbs/vbs.latin

```diff
@@ -4225,7 +4235,7 @@
 :de:de_-damn	are_vb
 :le:dedecet
 :vs:de_-de^c	conj2
```

Comment:
should we also have `conj2 3rd` here? or can we have other forms here?

Reply:
updated with 3rd for both and added L&S rare personal forms